src/Sulu/Bundle/AdminBundle/Resources/public/js/aura_extensions/default-extension.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define(['services/husky/util', 'services/husky/translator'], function(util, translator) {
```

Comment:
1. I would not inject the util class like this, but use the extension (what else do we have the extension for?) Makes sense for me for stuff like this, because we need the utils in almost every component.
2. Until now `util` and `translator` would have been capitalized, I know that you are doing that because these aren't constructors, but then we should write that down somewhere, and theoretical we would have to update a lot of files, because it has never been done this way (at least I haven't seen it).